.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```

Comment:
Just saying, I think we've decided to use npm instead of yarn to avoid dependency conflicts

Reply:
Ok!